### ISSUE TITLE
issue on v1.10.4

### ISSUE THREAD

**Issue by johnkalangha** [osTicket/osTicket#5415]
![image](https://user-images.githubusercontent.com/30627667/76812438-ce2c9000-682f-11ea-96c8-e7fecb2fb22f.png)

Hi, anyone can help me, we have 2 teams in 1 department, my problem is when the user creates a ticket both teams can see the ticket (MY ticket tab) even if the auto assign ticket is for team2, anyone can help me? 

note im still preparing my osticket for v1.14.4 but this issue is badly needed.. thank you and cheers

**Comment by JediKev** [comment]
@johnkalangha 

This sounds like a bug that was fixed in later updates. First, you are using a really old and outdated version that we don't even provide support for anymore (1.12.x is getting security updates and 1.14.x is receiving major updates, everything below 1.12.x is no longer supported). Secondly, you will need to upgrade to a supported version, either v1.12.5 or v1.14.1. Once you upgrade to a supported version, you can retest the issue. If it's still an issue with the supported versions then please, report back.

Cheers.

**Comment by johnkalangha** [comment]
yes, sir, I will update my version to 1.14.1 but regarding this issue I need to solve this in the same version, I'm still working on it to v 1.14.1 because of modification on client-side.

**Comment by JediKev** [comment]
@johnkalangha 

Well unfortunately you’ll have to debug yourself as that version is no longer supported. You could try upgrading to the latest release of the 1.10.x series (v1.10.7) to see if it was addressed in later maintenance updates.

Cheers.

**Comment by johnkalangha** [comment]
sir can is ask if there is a patch to this bug on v 1.10.4+ or any later so that can i find in relases tab. thanks
